### ISSUE TITLE
Still run audio when no device is currently available

### ISSUE THREAD

**Issue by def-** [ddnet/ddnet#4603]
One might get connected later ingame, DM by SPYRES:

> Hey Deen. Was wondering if this was intended in the client. I use wireless headphone, if they are not connected and dont have another audio playback device available, the client will say "The audio device cannot be initialised" which makes sense. However, I find that audio initialisation only happens at client start and making a playback available after client initialisation forces you to relaunch to get audio. Would it be possible to constantly check/refresh for available audio devices to prevent from relaunching the client constantly? Really minor, I mean there are ways around it in the sense that I could use another 'dummy' playback device, make that available so that when I do switch on wireless headphones there is no issue switching over as it would be my computers preferred default playback device.  Anyways, just thought I would let you know. More cosmetic/experience I guess.

**Comment by Robyt3** [comment]
Should be possible by checking for [`SDL_AUDIODEVICEADDED`](https://wiki.libsdl.org/SDL2/SDL_AudioDeviceEvent).

**Comment by AssassinTee** [comment]
@SollyBunny is this something you could look into as our audio expert now? 😅

**Comment by SollyBunny** [comment]
I don't think this is related to my stuff, I think I could do it. This has affected me before
